superset/base/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is there any specific reason why the labels are specified in each resource rather than in kustomization.yaml?

Reply:
Turns out that a side-effect of "commonLabels" is that the services also wind-up with the labels and that has a negative effect when mapping a route to a service/pod.  https://github.com/kubernetes-sigs/kustomize/issues/1009  I was not able to get the metadataLabelTransformer workaround to work.

Reply:
I have created a tracking issue to remind us (me) to go back and use the labelTransformer when we have a released kfctl that supports it.  https://github.com/opendatahub-io/odh-manifests/issues/7